insights/parsers/mongod_conf.py

```diff
@@ -48,9 +50,9 @@
 Examples:
 
     >>> mongod_conf = shared[MongodConf]
-    >>> 'storage' in mongod_conf.data
+    >>> 'storage' in mongod_conf
     True
-    >>> mongod_conf.data['storage']['dbPath']
+    >>> mongod_conf['storage']['dbPath']
```

Comment:
As per my env, it's `dbpath` but not `dbPath`, it's all lower case letter. Better cast them to lower case forcibly.